test-data/unit/check-warnings.test

```diff
@@ -234,3 +234,186 @@ def foo(a1: A) -> int:
 class A:
     def __init__(self, x: int) -> None:
         self._x = x
+
+[case testUninitializedAttribute]
+# flags: --warn-uninitialized-attributes
+class A:
+    x: int
+a = A() # E: Cannot instantiate abstract class "A" with abstract attribute "x"
```

Comment:
I now went with `Class "A" has annotated but unset attributes: "x"` because "Cannot instantiate" seemed too strong to me (because the instantiation will work fine at runtime). But I'm not attached to this at all and I'm happy to change it again.